### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/tooot-app-app/commit/7be6c79a065586d3623b3d8b6a0e31c254db88ec#commitcomment-89322862

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/tooot-app-app/commit/be3af25b4a5b73762673c262f9119a11eaaa53e9

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: src/startup/netInfo.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ const netInfo = async (): Promise<{
       if (resVerify.id !== instance.account.id) {
         log('error', 'netInfo', 'local id does not match remote id')
         store.dispatch(removeInstance(instance))
-        return Promise.resolve({ connected: true, corruputed: '' })
+        return Promise.resolve({ connected: true, corrupted: '' })
```
Comment (jsoref) [inline src/startup/netInfo.ts:54]: ??

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine.

File: android/app/build.gradle [hunk 2]
```diff
@@ -314,7 +314,7 @@ dependencies {
 if (isNewArchitectureEnabled()) {
     // If new architecture is enabled, we let you build RN from source
     // Otherwise we fallback to a prebuilt .aar bundled in the NPM package.
-    // This will be applied to all the imported transtitive dependency.
```
Comment (xmflsct) [inline android/app/build.gradle:317]: 😂 this is fun. such comment was copied directly from the react-native repo

**Comment by xmflsct** [comment]
Hahaha, I could not stop laughing of so many spelling failures I have made. 😂 Also did not know such spelling check possibility existed!

File: android/app/build.gradle (hunk 2 above)
Comment (jsoref) [reply android/app/build.gradle:317]: Oh, my.... https://cs.github.com/?scopeName=All+repos&scope=&q=%22This+will+be+applied+to+all+the+imported+transtitive+dependency.%22

https://github.com/ReactNativeSchool/react-native-firebase-chat-app/ ?

Comment (xmflsct) [reply android/app/build.gradle:317]: https://github.com/react-native-community/rn-diff-purge/blob/version/0.70.5/RnDiffApp/android/app/build.gradle#L285 Basically this particular phrase appeared in many versions tagged in this repo where people copied from

Comment (jsoref) [reply android/app/build.gradle:317]: Added to https://github.com/stars/jsoref/lists/to-check-spelling (note that the list mostly grows, and is around 300 items long)...

File: src/startup/netInfo.ts (hunk 1 above)
Comment (xmflsct) [reply src/startup/netInfo.ts:54]: Interesting, due to this typo, the value was never used!

**Review by xmflsct — APPROVED** [review]

**Pull request merged**